CHANGELOG.md

```diff
@@ -5,6 +5,92 @@ All notable changes to this project will be documented in this file.
+- Allow backslash, carriage return in PQL strings ([#1713](https://github.com/pilosa/pilosa/pull/1713))
+- Add base system, curl and jq for debug and checks. ([#1707](https://github.com/pilosa/pilosa/pull/1707))
+- Add "Rows" and "GroupBy" functionality ([#1647](https://github.com/pilosa/pilosa/pull/1647))
+- Add `clear` functional option for imports ([#1699](https://github.com/pilosa/pilosa/pull/1699))
+- Maintain Available Shards ([#1600](https://github.com/pilosa/pilosa/pull/1600)), ([#1695](https://github.com/pilosa/pilosa/pull/1695)), ([#1624](https://github.com/pilosa/pilosa/pull/1624)), ([#1663](https://github.com/pilosa/pilosa/pull/1663))
+- Adds missing rowID/Key columnID/Key tests ([#1683](https://github.com/pilosa/pilosa/pull/1683))
+- Store() function ([#1666](https://github.com/pilosa/pilosa/pull/1666))
+- Adds diagnostics CPUArch field ([#1671](https://github.com/pilosa/pilosa/pull/1671))
+- Add CircleCI step to generate Docker image and push to Docker hub ([#1673](https://github.com/pilosa/pilosa/pull/1673))
+- Implement ClearRow() query ([#1645](https://github.com/pilosa/pilosa/pull/1645))
+- Add support for Bool fields ([#1658](https://github.com/pilosa/pilosa/pull/1658))
+- Make translate map size configurable ([#1653](https://github.com/pilosa/pilosa/pull/1653))
+- Adds DirectAdd function to roaring.Bitmap ([#1646](https://github.com/pilosa/pilosa/pull/1646))
+- Roaring import ([#1622](https://github.com/pilosa/pilosa/pull/1622))
+- Add Not() query ([#1635](https://github.com/pilosa/pilosa/pull/1635))
+- Implements Options call and excludeRowAttrs, excludeColumns, columnAttrs and shards args ([#1631](https://github.com/pilosa/pilosa/pull/1631))
+- Added field options to pilosa import ([#1625](https://github.com/pilosa/pilosa/pull/1625))
+- Existence Tracking ([#1788](https://github.com/pilosa/pilosa/pull/1788)), ([#1672](https://github.com/pilosa/pilosa/pull/1672)), ([#1628](https://github.com/pilosa/pilosa/pull/1628))
+
+### Changed
+
+- Convert the anti-entropy logic to use `ImportRoaring` instead of `QueryNode` ([#1780](https://github.com/pilosa/pilosa/pull/1780))
+- Simplify "require-*" logic in Makefile ([#1755](https://github.com/pilosa/pilosa/pull/1755))
+- Logging cleanup ([#1748](https://github.com/pilosa/pilosa/pull/1748))
+- Improve benchmarking and performance. ([#1741](https://github.com/pilosa/pilosa/pull/1741))
+- Remove TravisCI, add CircleCI shield ([#1740](https://github.com/pilosa/pilosa/pull/1740))
+- Upgrade Peg dependency and regenerate grammar ([#1725](https://github.com/pilosa/pilosa/pull/1725))
+- Upgrade to protoc 3.6.1. (also updated protoc-gen-gofast). ([#1724](https://github.com/pilosa/pilosa/pull/1724))
+- Move column attrs logic to executor ([#1677](https://github.com/pilosa/pilosa/pull/1677))
+- Shrank n(container bit count cache) to int32 ([#1664](https://github.com/pilosa/pilosa/pull/1664))
+- Removing bounds check ([#1619](https://github.com/pilosa/pilosa/pull/1619))
```

Comment:
"Removing" breaks convention. This could also be placed in a `Performance` section as I have with other releases. Also would be good to add more context, e.g. "...to optimize roaring performance".